.github/workflows/build_all.yml

```diff
@@ -88,8 +88,19 @@ jobs:
             --env "IREE_WRITE_REMOTE_CCACHE=${IREE_WRITE_REMOTE_CCACHE}" \
             --env "CCACHE_NAMESPACE=gcr.io/iree-oss/base@sha256:61e9aae211007dbad95e1f429e9e5121fd5968c204791038424979c21146cf75" \
             gcr.io/iree-oss/base@sha256:61e9aae211007dbad95e1f429e9e5121fd5968c204791038424979c21146cf75 \
-            ./build_tools/cmake/build_all.sh \
-            "${BUILD_DIR}"
+            ./build_tools/cmake/build_all.sh "${BUILD_DIR}"
```

Comment:
FYI: build_all.yml is also used in `benchmark_large.yml`, so this will cause tests also run in that workflow

https://github.com/openxla/iree/blob/main/.github/workflows/benchmark_large.yml#L49

Reply:
Ah right... I touched those files a bit yesterday, but they should really be switched off of source builds entirely. Not sure how much energy I'll put into that myself though - need to get the core workflows into better shape before cleaning up the special ones.